docs/source/ec2_deployment.rst

```diff
@@ -0,0 +1,172 @@
+Using EC2 for Docker Based Deployments
+======================================
+
+The ``dallinger`` commandline tool has some helpers for managing EC2 instances
+to help facilitate using docker to deploy experiments on those instances (see
+:doc:`docker_support`).
+
+
+Initial Setup
+-------------
+
+Route53 DNS
+~~~~~~~~~~~
+
+In order to use EC2 provisioning, you must have a domain created in AWS Route53
+managed using the same credentials you use for your experiment (see:
+:doc:`aws_etc_keys`). The examples below assume that you have setup a domain
+``my-experiments.org`` in Route53.
```

Comment:
In the Psynet documentation we had the following text, perhaps add something like this here:

"If your lab is doing this for the first time, you probably need to acquire a domain name for your experiment server. This is the parent URL that will be used to host your experiments. If your lab already has a domain name, you can skip this step. On the AWS online console, navigate to the Route 53 service. On the Dashboard you can register a domain name. Note that different domain names come with different costs, and that registering a domain name can take from a few minutes to several hours. Before proceeding with the next steps, please wait until the AWS console tells you that the registration is complete."

This is taken from here were step 15 represent this step: https://psynetdev.gitlab.io/PsyNet/deploy/aws_server_setup.html